src/graphile-worker/enums/graphile-worker-pattern.ts

```diff
@@ -15,4 +15,5 @@ export enum GraphileWorkerPattern {
   UPSERT_BLOCK_MINED_EVENT = 'UPSERT_BLOCK_MINED_EVENT',
   UPSERT_DEPOSIT = 'UPSERT_DEPOSIT',
   REFRESH_POOL_4_POINTS = 'REFRESH_POOL_4_POINTS',
+  UPDATE_RANKS = 'UPDATE_RANKS',
```

Comment:
nit: to be more explicit
```suggestion
  UPDATE_USER_RANKS = 'UPDATE_USER_RANKS',
```